kind: add page (#4394)
tldr-pages/tldr: pages/common/kind.md

# kind

> > Kind is a tool for running local Kubernetes clusters using Docker container "nodes".
> > Kind was primarily designed for testing Kubernetes itself, but may be used for local development or CI.

Reviewer: Maybe we could have this more concise?
```suggestion
> Tool for running local Kubernetes clusters using Docker container "nodes".
> Designed for testing Kubernetes itself, but may be used for local development or continuous integration.
```

> More information: <https://github.com/kubernetes-sigs/kind>.

- Create a local Kubernetes cluster:

`kind create cluster --name {{cluster-name}}`

- Delete one or more clusters:

`kind delete clusters {{cluster-name}}`

> - Get one of [clusters, nodes, kubeconfig]:
> 
> `kind get clusters`

Reviewer: How does one determine which one to fetch? If it's by the keyword there, what about this?
```suggestion
- Get details about clusters, nodes, or the kubeconfig:

`kind get {{clusters|nodes|kubeconfig}}`
```

> - Export one of [kubeconfig, logs]:
> 
> `kind export logs`

Reviewer: Same here.
```suggestion
- Export the kubeconfig or the logs:

`kind export {{kubeconfig|logs}}`
```
